Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rust:1.63 as builder
```

Comment:
Latest release is 1.64

Reply:
In general is always better to choose a specific version instead of "latest" in order to have reproducible build. However this PR is on hold.